src/Compiler/ES/Codegen.idr

```diff
@@ -145,6 +145,7 @@ applyCon NOTHING _ [] = "{h" <+> softColon <+> "0}"
 applyCon CONS    _ as = applyObj (conTags as)
 applyCon JUST    _ as = applyObj (conTags as)
 applyCon RECORD  _ as = applyObj (conTags as)
+applyCon UNIT    _ [] = "undefined"
```

Comment:
For consistency, we should do the same in pattern matches (switch statements), even if we hopefully will never need it.